AnimationSystem/AnimationSystem.cc

```diff
@@ -0,0 +1,1074 @@
+          else if (boneName == "Left_littleFinger2") boneIndexes.Left_littleFinger2 = i;
+          else if (boneName == "Left_littleFinger3") boneIndexes.Left_littleFinger3 = i;
+          else if (boneName == "Right_shoulder") boneIndexes.Right_shoulder = i;
+          else if (boneName == "Right_arm") boneIndexes.Right_arm = i;
+          else if (boneName == "Right_elbow") boneIndexes.Right_elbow = i;
+          else if (boneName == "Right_wrist") boneIndexes.Right_wrist = i;
+          else if (boneName == "Right_middleFinger1") boneIndexes.Right_middleFinger1 = i;
+          else if (boneName == "Right_middleFinger2") boneIndexes.Right_middleFinger2 = i;
+          else if (boneName == "Right_middleFinger3") boneIndexes.Right_middleFinger3 = i;
+          else if (boneName == "Right_thumb0") boneIndexes.Right_thumb0 = i;
+          else if (boneName == "Right_thumb1") boneIndexes.Right_thumb1 = i;
+          else if (boneName == "Right_thumb2") boneIndexes.Right_thumb2 = i;
+          else if (boneName == "Right_indexFinger1") boneIndexes.Right_indexFinger1 = i;
+          else if (boneName == "Right_indexFinger2") boneIndexes.Right_indexFinger2 = i;
+          else if (boneName == "Right_indexFinger3") boneIndexes.Right_indexFinger3 = i;
+          else if (boneName == "Right_ringFinger1") boneIndexes.Right_ringFinger1 = i;
+          else if (boneName == "Right_ringFinger2") boneIndexes.Right_ringFinger2 = i;
+          else if (boneName == "Right_ringFinger3") boneIndexes.Right_ringFinger3 = i;
+          else if (boneName == "Right_littleFinger1") boneIndexes.Right_littleFinger1 = i;
+          else if (boneName == "Right_littleFinger2") boneIndexes.Right_littleFinger2 = i;
+          else if (boneName == "Right_littleFinger3") boneIndexes.Right_littleFinger3 = i;
+          else if (boneName == "Right_leg") boneIndexes.Right_leg = i;
+          else if (boneName == "Right_knee") boneIndexes.Right_knee = i;
+          else if (boneName == "Right_ankle") boneIndexes.Right_ankle = i;
+          else if (boneName == "Right_toe") boneIndexes.Right_toe = i;
+          else if (boneName == "Left_leg") boneIndexes.Left_leg = i;
+          else if (boneName == "Left_knee") boneIndexes.Left_knee = i;
+          else if (boneName == "Left_ankle") boneIndexes.Left_ankle = i;
+          else if (boneName == "Left_toe") boneIndexes.Left_toe = i;
+        }
+      }
+
+      // -------------------------------------------------------------------------
+
+      jsonStr += "[";
+      for (unsigned int i = 0; i < declarations.size(); i++) {
+        AnimationGroupDeclaration declaration = declarations[i];
+        std::vector<Animation *> animationGroup;
+
+        jsonStr += "{";
+        jsonStr += "\"name\":";
+        jsonStr += "\"" + declaration.groupName + "\"";
+        jsonStr += ",";
+        jsonStr += "\"index\":";
+        jsonStr += std::to_string(declaration.index);
```

Comment:
I use `to_string` to convert cpp `int` type to cpp `string` type so it can be added into `jsonStr`.
Not to define `string` type in JSON.
So the result is correctly `"index":0`, not `"index":"0"`.